packages/lab/stories/multiline-input/multiline-input.doc.stories.mdx

```diff
@@ -72,6 +72,8 @@ You can set Multiline Input to a disabled state, with no resultant action when t
   <Story id="lab-multiline-input--disabled" />
 </Canvas>
 
+- When setting Multiline Input to disabled state, ensure to also disable any nested interactive adornments, such as buttons.
```

Comment:
This sentence sounds a bit clunky to me, but content can review this later

Reply:
cc. @sion-docs / @mikearildbrown